staging/prometheus-operator/patch/README.md

```diff
@@ -0,0 +1,9 @@
+# Fork Patches
```

Comment:
this is handy 🙌would be great to eventually get our other enhancements into here for ease of upgrades